Formula/xxhash.rb

```diff
@@ -11,13 +11,29 @@ class Xxhash < Formula
     sha256 "8d7d413a0165a18c9df428d40fe7ca88c7f841e318f594c29d3daf878fc9c484" => :el_capitan
   end
 
+  option "without-xxhsum", "Omits xxhsum executable"
+
   def install
     system "make"
-    bin.install "xxhsum"
+
+    if build.with? "xxhsum"
```

Comment:
per https://github.com/Homebrew/homebrew-core/pull/23440#issuecomment-362270097, we can replace this and  everything below with:

```ruby
system "make", "install", "PREFIX=#{prefix}"
```

Reply:
Oh, and the `sha256` line above should have `revision 1` added below it, to tell our CI to rebuild bottles containing the library:

```ruby
sha256 "4570ccd111df6b6386502791397906bf69b7371eb209af7d41debc2f074cdb22"
revision 1
```